include/lsst/afw/geom/SpherePoint.h

```diff
@@ -225,12 +223,12 @@ class SpherePoint final {
      */
 
     /**
-     * `true` if two points represent the same position.
+     * `true` if two points have the same longitude (ignoring wrap) and latitude.
```

Comment:
Sorry, but "ignoring wrap" is kind of confusing (does "ignore" mean that 90 degrees and 450 degrees are considered the same, or not?). It also seems redundant, given that `getLongitude()` is guaranteed to return a wrapped angle or NaN.

Reply:
I'm feel that I am in a Catch-22. The way I see it is:
- If we assume that the constructor wraps longitude on construction (which you complained about when I documented that), then it is unambiguous what equal longitude means in `operator==`.
- If we do not assume that the constructor wraps longitude on construction, then `operator==` is ambiguous, in that it is not clear if it compares the actual longitude or the wrapped version returned by `getLongitude()`

I personally prefer to document that longitude is wrapped on construction. That neatly explains why `+/-inf` is turned into `nan` and gives an the user a clear model of what is going on which can be extended to methods such as `operator==`. But I can see why you felt that exposed the implementation too much. If you still feel that way then please suggest clear and unambiguous documentation for `operator==`

Reply:
How about removing the parenthetical note from the title, but writing:

> `@returns true if this point has exactly the same values of `getLongitude()` and `getLatitude()` as `other`, false otherwise`

(Also, I just noticed while looking at this that `@note` is in front of the params/returns sections; it should go next to the warning).

Reply:
OK. I moved one other note as well